universal-redux example broken

Steps To Reproduce
======
I'm guessing the solution for both these problems centers around a correct .babelrc in the universal-redux folder

Problem 1
------
```
$ git clone 4Catalyzer/found
$ cd found/examples/universal-redux
$ npm install
$ npm start
$ babel-node src/server
/Users/florian.mettetal/Code/.private/found/examples/universal-redux/node_modules/babel-core/lib/transformation/file/options/option-manager.js:180
          throw new ReferenceError(messages.get("pluginUnknown", plugin, loc, i, dirname));
          ^

ReferenceError: Unknown plugin "dev-expression" specified in "/Users/florian.mettetal/Code/.private/found/.babelrc" at 0, attempted to resolve relative to "/Users/florian.mettetal/Code/.private/found"
    at /Users/florian.mettetal/Code/.private/found/examples/universal-redux/node_modules/babel-core/lib/transformation/file/options/option-manager.js:180:17
    at Array.map (native)
    at Function.normalisePlugins (/Users/florian.mettetal/Code/.private/found/examples/universal-redux/node_modules/babel-core/lib/transformation/file/options/option-manager.js:158:20)
    at OptionManager.mergeOptions (/Users/florian.mettetal/Code/.private/found/examples/universal-redux/node_modules/babel-core/lib/transformation/file/options/option-manager.js:234:36)
    at OptionManager.init (/Users/florian.mettetal/Code/.private/found/examples/universal-redux/node_modules/babel-core/lib/transformation/file/options/option-manager.js:368:12)
    at compile (/Users/florian.mettetal/Code/.private/found/examples/universal-redux/node_modules/babel-register/lib/node.js:103:45)
    at loader (/Users/florian.mettetal/Code/.private/found/examples/universal-redux/node_modules/babel-register/lib/node.js:144:14)
    at Object.require.extensions.(anonymous function) [as .js] (/Users/florian.mettetal/Code/.private/found/examples/universal-redux/node_modules/babel-register/lib/node.js:154:7)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
error Command failed with exit code 1.
```

Problem 2
------
```
$ git clone 4Catalyzer/found
$ cd found/
$ rm .babelrc
$ cd examples/universal-redux
$ npm install
$ npm start
$ babel-node src/server
SyntaxError: /Users/florian.mettetal/Code/.private/ultimate-react/src/server.js: Unexpected token (86:6)
  84 |     .status(renderArgs.error ? renderArgs.error.status : 200)
  85 |     .send(renderPageToString(
> 86 |       <Provider store={store}>
     |       ^
  87 |         <RouterProvider router={renderArgs.router}>
  88 |           {render(renderArgs)}
  89 |         </RouterProvider>
    at Parser.pp$5.raise (/Users/florian.mettetal/Code/.private/ultimate-react/node_modules/babylon/lib/index.js:4454:13)
```

When I made this issue I was using yarn19, npm4 - I've upgraded everything to latest (yarn24.6, npm5) and restarted, did a fresh clone, top of my git history is 8dcd8

```
~/Code//found/examples/universal-redux (master●)
$ npm start

> universal-redux@0.1.0 start /Users/florian.mettetal/Code/.private/found/examples/universal-redux
> babel-node src/server

/Users/florian.mettetal/Code/.private/found/examples/universal-redux/node_modules/babel-core/lib/transformation/file/options/option-manager.js:180
          throw new ReferenceError(messages.get("pluginUnknown", plugin, loc, i, dirname));
          ^

ReferenceError: Unknown plugin "dev-expression" specified in "/Users/florian.mettetal/Code/.private/found/.babelrc" at 0, attempted to resolve relative to "/Users/florian.mettetal/Code/.private/found"
    at /Users/florian.mettetal/Code/.private/found/examples/universal-redux/node_modules/babel-core/lib/transformation/file/options/option-manager.js:180:17
    at Array.map (native)
    at Function.normalisePlugins (/Users/florian.mettetal/Code/.private/found/examples/universal-redux/node_modules/babel-core/lib/transformation/file/options/option-manager.js:158:20)
    at OptionManager.mergeOptions (/Users/florian.mettetal/Code/.private/found/examples/universal-redux/node_modules/babel-core/lib/transformation/file/options/option-manager.js:234:36)
    at OptionManager.init (/Users/florian.mettetal/Code/.private/found/examples/universal-redux/node_modules/babel-core/lib/transformation/file/options/option-manager.js:368:12)
    at compile (/Users/florian.mettetal/Code/.private/found/examples/universal-redux/node_modules/babel-register/lib/node.js:103:45)
    at loader (/Users/florian.mettetal/Code/.private/found/examples/universal-redux/node_modules/babel-register/lib/node.js:144:14)
    at Object.require.extensions.(anonymous function) [as .js] (/Users/florian.mettetal/Code/.private/found/examples/universal-redux/node_modules/babel-register/lib/node.js:154:7)
    at Module.load (module.js:503:32)
    at tryModuleLoad (module.js:466:12)
```


**(same as before) After deleting .babelrc**
```
$ npm start

> universal-redux@0.1.0 start /Users/florian.mettetal/Code/.private/found/examples/universal-redux
> babel-node src/server

/Users/florian.mettetal/Code/.private/found/examples/universal-redux/node_modules/babel-core/lib/transformation/file/index.js:590
      throw err;
      ^

SyntaxError: /Users/florian.mettetal/Code/.private/found/examples/universal-redux/src/server.js: Unexpected token (86:6)
  84 |     .status(renderArgs.error ? renderArgs.error.status : 200)
  85 |     .send(renderPageToString(
> 86 |       <Provider store={store}>
     |       ^
  87 |         <RouterProvider router={renderArgs.router}>
  88 |           {render(renderArgs)}
  89 |         </RouterProvider>
    at Parser.pp$5.raise (/Users/florian.mettetal/Code/.private/found/examples/universal-redux/node_modules/babylon/lib/index.js:4454:13)
    at Parser.pp.unexpected (/Users/florian.mettetal/Code/.private/found/examples/universal-redux/node_modules/babylon/lib/index.js:1761:8)
    at Parser.pp$3.parseExprAtom (/Users/florian.mettetal/Code/.private/found/examples/universal-redux/node_modules/babylon/lib/index.js:3750:12)
    at Parser.pp$3.parseExprSubscripts (/Users/florian.mettetal/Code/.private/found/examples/universal-redux/node_modules/babylon/lib/index.js:3494:19)
    at Parser.pp$3.parseMaybeUnary (/Users/florian.mettetal/Code/.private/found/examples/universal-redux/node_modules/babylon/lib/index.js:3474:19)
    at Parser.pp$3.parseExprOps (/Users/florian.mettetal/Code/.private/found/examples/universal-redux/node_modules/babylon/lib/index.js:3404:19)
    at Parser.pp$3.parseMaybeConditional (/Users/florian.mettetal/Code/.private/found/examples/universal-redux/node_modules/babylon/lib/index.js:3381:19)
    at Parser.pp$3.parseMaybeAssign (/Users/florian.mettetal/Code/.private/found/examples/universal-redux/node_modules/babylon/lib/index.js:3344:19)
    at Parser.pp$3.parseExprListItem (/Users/florian.mettetal/Code/.private/found/examples/universal-redux/node_modules/babylon/lib/index.js:4312:16)
    at Parser.pp$3.parseCallExpressionArguments (/Users/florian.mettetal/Code/.private/found/examples/universal-redux/node_modules/babylon/lib/index.js:3573:20)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! universal-redux@0.1.0 start: `babel-node src/server`
npm ERR! Exit status 1
npm ERR!
```

--- a/examples/universal-redux/.babelrc
+++ b/examples/universal-redux/.babelrc
@@ -1,0 +1,8 @@
+{
+  "presets": [
+    ["latest", {
+      "es2015": { "loose": true }
+    }],
+    "react"
+  ]
+}

--- a/examples/universal-redux/package.json
+++ b/examples/universal-redux/package.json
@@ -8,6 +8,7 @@
   "dependencies": {
     "babel-cli": "^6.24.1",
     "babel-loader": "^7.0.0",
+    "babel-polyfill": "^6.23.0",
     "babel-preset-latest": "^6.24.1",
     "babel-preset-react": "^6.24.1",
     "express": "^4.15.3",

--- a/examples/universal-redux/src/server.js
+++ b/examples/universal-redux/src/server.js
@@ -2,7 +2,6 @@ import express from 'express';
 import { Actions as FarceActions, ServerProtocol } from 'farce';
 import { getStoreRenderArgs, resolver, RedirectException } from 'found';
 import { RouterProvider } from 'found/lib/server';
-import path from 'path';
 import React from 'react';
 import ReactDOMServer from 'react-dom/server';
 import { Provider } from 'react-redux';
@@ -17,10 +16,13 @@ const PORT = 3000;
 const app = express();
 
 const webpackConfig = {
-  entry: './src/client',
+  entry: [
+    'babel-polyfill',
+    './src/client',
+  ],
 
   output: {
-    path: path.resolve('build'),
+    path: '/',
     publicPath: '/static',
     filename: 'bundle.js',
   },
@@ -38,7 +40,7 @@ function renderPageToString(element, state) {
 <html>
 
 <head>
-  <meta charset="UTF-8">
+  <meta charset="utf-8">
   <title>Found Universal Redux Example</title>
 </head>
 
@@ -55,8 +57,8 @@ function renderPageToString(element, state) {
 }
 
 app.use(webpackMiddleware(webpack(webpackConfig), {
-  noInfo: true,
   publicPath: webpackConfig.output.publicPath,
+  stats: { colors: true },
 }));
 
 app.use(async (req, res) => {

--- a/examples/universal/.babelrc
+++ b/examples/universal/.babelrc
@@ -1,0 +1,8 @@
+{
+  "presets": [
+    ["latest", {
+      "es2015": { "loose": true }
+    }],
+    "react"
+  ]
+}

--- a/examples/universal/package.json
+++ b/examples/universal/package.json
@@ -8,6 +8,7 @@
   "dependencies": {
     "babel-cli": "^6.24.1",
     "babel-loader": "^7.0.0",
+    "babel-polyfill": "^6.23.0",
     "babel-preset-latest": "^6.24.1",
     "babel-preset-react": "^6.24.1",
     "express": "^4.15.3",

--- a/examples/universal/src/server.js
+++ b/examples/universal/src/server.js
@@ -1,6 +1,5 @@
 import express from 'express';
 import { getFarceResult } from 'found/lib/server';
-import path from 'path';
 import ReactDOMServer from 'react-dom/server';
 import webpack from 'webpack';
 import webpackMiddleware from 'webpack-dev-middleware';
@@ -13,10 +12,13 @@ const PORT = 3000;
 const app = express();
 
 const webpackConfig = {
-  entry: './src/client',
+  entry: [
+    'babel-polyfill',
+    './src/client',
+  ],
 
   output: {
-    path: path.resolve('build'),
+    path: '/',
     publicPath: '/static',
     filename: 'bundle.js',
   },
@@ -29,8 +31,8 @@ const webpackConfig = {
 };
 
 app.use(webpackMiddleware(webpack(webpackConfig), {
-  noInfo: true,
   publicPath: webpackConfig.output.publicPath,
+  stats: { colors: true },
 }));
 
 app.use(async (req, res) => {
@@ -50,7 +52,7 @@ app.use(async (req, res) => {
 <html>
 
 <head>
-  <meta charset="UTF-8">
+  <meta charset="utf-8">
   <title>Found Universal Example</title>
 </head>
 

